### PR TITLE
Add Girder to file load menu

### PR DESCRIPTION
**Pull request by HastingsGreer**
TODO:
Store file provenance with datasets
enable download of files which user has private access to

### CODE REVIEW

**Comment by jourdain** [comment]
I know it is a bit early but if you plan to keep your history, you should make sure you follow the semantic-release format for the commit messages.

File: build/webpack.base.config.js [hunk 1]
```diff
@@ -73,6 +77,7 @@ module.exports = {
         test: /\.(js|vue)$/,
         loader: 'eslint-loader',
         enforce: 'pre',
+        options: {ix:true}
```
Comment (floryst) [inline build/webpack.base.config.js]: Should be "fix: true". Didn't know about the autofix feature, so this will be useful!

Comment (HastingsGreer) [reply build/webpack.base.config.js]: Didn't mean to commit this! It doesn't mesh well with all workflows, so it should probably stay in m local work.

**Comment by floryst** [comment]
Can you grep for "http://" inside package-lock.json? There's one URL that should be changed to "https://".

File: src/components/core/GirderBox/script.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+import { Authentication as GirderAuthentication } from '@girder/components/src/components';
+import { FileManager as GirderFileManager } from '@girder/components/src/components/Snippet';
+import { Upload as GirderUpload } from '@girder/components/src/utils';
+
+import writeImageArrayBuffer from 'itk/writeImageArrayBuffer';
+import Matrix from 'itk/Matrix';
+
+// ----------------------------------------------------------------------------
+
+function convertVtkToItkImage(vtkImage) {
```
Comment (floryst) [inline src/components/core/GirderBox/script.js]: Can you use vtk.js' `ITKHelper.convertVtkToItkImage` here instead?

File: src/components/core/GirderBox/script.js [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+    },
+    ...mapState({
+      proxyManager: 'proxyManager',
+    }),
+  },
+  mounted() {
+    this.$root.$on('girder_upload_proxy', (proxyId) => {
+      this.upload(proxyId);
+    });
+    console.log('mounted');
```
Comment (floryst) [inline src/components/core/GirderBox/script.js]: Remove console.log

File: src/components/core/GirderBox/script.js [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+          /* eslint-disable-next-line no-underscore-dangle */
+          url: `${this.girderRest.apiRoot}/item/${elem._id}/download`,
+          name: elem.name,
+        };
+      });
+
+      this.$store.dispatch('files/openRemoteFiles', rfiles);
+      // this.$emit('close');
+    },
+    export2pc(proxyId) {
```
Comment (floryst) [inline src/components/core/GirderBox/script.js]: Is this function used, or was it for debugging purposes?

Comment (HastingsGreer) [reply src/components/core/GirderBox/script.js]: It still needs to be wired to a ui element: it allows downloading a dataset as a .nrrd file which is not currently supported in paraview glance

**Pull request merged**